fesvr/htif.cc

```diff
@@ -138,7 +138,12 @@ std::map<std::string, uint64_t> htif_t::load_payload(const std::string& payload,
 
 void htif_t::load_program()
 {
-  std::map<std::string, uint64_t> symbols = load_payload(targs[0], &entry);
+  bool is32bit = false;
+  std::map<std::string, uint64_t> symbols = load_payload(targs[0], &entry, is32bit);
+  if(is32bit != (32 == max_xlen)){
+    printf("mixing 32bit with 64bit payload!\n");
```

Comment:
This is an unclear error message; exactly what is being mixed?

Can’t we just delete this code altogether and only have the one error message below?